.github/workflows/python_cibuildwheel.yml

```diff
@@ -0,0 +1,233 @@
+name: Python cibuildwheel
+
+on:
+  push:
+    branches: [ master, develop, actions_pypi ]
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  python_bindings:
+    name: Build wheel for cp${{ matrix.python }}-${{ matrix.platform_id }}-${{ matrix.manylinux_image }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.allow_failure }}
+    strategy:
+      # Ensure that a wheel builder finishes even if another fails
+      fail-fast: false
+      matrix:
+        include:
+          # Window 64 bit
+          # Note: windows-2019 is needed for older Python versions:
+          # https://github.com/scikit-learn/scikit-learn/issues/22530
+          - os: windows-2019
+            python: 38
+            bitness: 64
+            platform_id: win_amd64
+            cmake_compiler: vc16
+            arch: x64
+            allow_failure: false
+          - os: windows-latest
+            python: 39
+            bitness: 64
+            cmake_compiler: vc17
+            platform_id: win_amd64
+            arch: x64
+            allow_failure: false
+          - os: windows-latest
+            python: 310
+            bitness: 64
+            platform_id: win_amd64
+            cmake_compiler: vc17
+            arch: x64
+            allow_failure: false
+
+          # Window 32 bit
+          - os: windows-latest
+            python: 38
+            bitness: 32
+            platform_id: win32
+            cmake_compiler: vc17
+            arch: x86
+            allow_failure: true
+          - os: windows-latest
+            python: 39
+            bitness: 32
+            platform_id: win32
+            cmake_compiler: vc17
+            arch: x86
+            allow_failure: true
```

Comment:
These two I can't get to work

Reply:
I think that's ok, I had issues myself with 32-bit wheels. 32-bit windows is basically obsolete.